msteams-platform/concepts/build-and-test/teams-developer-portal.md

```diff
@@ -9,6 +9,33 @@ ms.author: surbhigupta
 
 # Manage your Teams apps using Developer Portal
 
+> [!IMPORTANT]
+> As you have migrated from App Studio to Developer Portal, the following table provides the detailed information of the features supported in the Developer Portal:
```

Comment:
"As you have migrated..." doesnt make much sense here.

Please move the entire table to the AppStudio page again, and but a banner something along the lines of> "Why you should start using Developer Portal instead.."